python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1011,7 +1018,33 @@ def _transform_arg(
         # use __reduce_ex__ instead...
         if type(arg) is tuple:
             # Must come first to avoid infinite recursion
-            return tuple(_transform_arg(a, attribute_name, seen) for a in arg)
+            if (
+                len(arg) > 0
+                and isinstance(arg[0], _MethodProxy)
+                and arg[0].__customqualname__ in _SPECIAL_FUNCTIONS_ARGS_MAP
+            ):
+                indices_map = _SPECIAL_FUNCTIONS_ARGS_MAP[
+                    arg[0].__customqualname__
+                ]
+                method_proxy, cust_args, cust_kwargs = arg
```

Comment:
I don't like `cust` as a prefix to `cust_args` here. "Custom" is too general of a word and I think we can choose a more precise/meaningful name. I am proposing `original_args` because it's pre-transformation. Other options: `input_args` or `untransformed_args`, or suggest your own.
```suggestion
                method_proxy, original_args, original_kwargs = arg
```